bin/move_it_mirror.py

```diff
@@ -43,22 +43,22 @@ class MoveItMirror(MoveItBase):
 
     def __init__(self, cmd_args):
         """Set up the mirror."""
-        super(MoveItMirror, self).__init__(cmd_args, "mirror")
-        self.cache_lock = Lock()
         LOGGER.info("Starting publisher on port %s.", str(cmd_args.port))
-        self.pub = Publisher("tcp://*:" + str(cmd_args.port), "move_it_mirror")
+        publisher = Publisher("tcp://*:" + str(cmd_args.port), "move_it_mirror")
+        super(MoveItMirror, self).__init__(cmd_args, "mirror", publisher=publisher)
+        self.cache_lock = Lock()
 
     def reload_cfg_file(self, filename):
         """Reload the config file."""
         reload_config(filename, self.chains, self.create_listener_notifier,
-                      MirrorRequestManager, self.pub)
+                      MirrorRequestManager, publisher=self.sync_publisher)
```

Comment:
This function and the next are the same, one should just call the other

Reply:
Done.